.github/actions/run-android-device-farm-test/action.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Run Android tests on Device Farm'
+inputs:
+  apk-path:
+    required: true
+  app-id:
+    required: true
+  project-arn:
+    required: true
+  device-pool-arn:
+    required: true
+  arguments:
+    default: _
+outputs:
+  test-results-path:
+    value: ${{ steps.get-test-results.outputs.results-path }}
+runs:
+  using: "composite"
+  steps:
+    - name: Run the tests
+      uses: realm/aws-devicefarm/test-application@d94e739490340474f8464f74289a0f48706dd3a3
+      id: run-tests
+      with:
+        project_arn: ${{ inputs.project-arn }}
+        device_pool_arn: ${{ inputs.device-pool-arn }}
```

Comment:
How do we configure the list of devices? E.g. if we want to run on more than one device?

Reply:
The list of devices is configured in AWS device farm, you can create different device pool that you reference by their `arn`. The issue with running on multiple devices is pulling resources out (logs etc.) and aggregating the results. The current .NET is also running on 1 device from the pool of selected device for simplicity